model/task_lifecycle.go

```diff
@@ -826,6 +826,9 @@ func getBuildStatus(buildTasks []task.Task) string {
 
 	// started but not finished
 	for _, t := range buildTasks {
+		if t.Status == evergreen.TaskStarted {
```

Comment:
Does it make sense to do the same for [versions/patches](https://github.com/evergreen-ci/evergreen/blob/4e70303913f203ba2d5340d1cb6801f32765d5ab/model/task_lifecycle.go#L918)? If so we wouldn't need to keep track of whether the build status has changed. I think?

Reply:
By "this" do you mean checking if build.Status == evergreen.BuildStarted for example? I don't think it's necessary, and I like casing on the build status because then we don't do DB lookups that we don't need.